### DISCUSSION TITLE
Round of Applause

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#233] [General]
Let's just remember, that this guy used his hard earned money to develop this app, which was in development for a few years. Every purchase of this app will help develop it further, until the day it ultimately becomes an industry standard app that people can use if they don't have Toon Boom Harmony or Adobe Animate. So, for all the hard work Keiwando has done, I say we all give him a round of applause.

Buy ToonSquid, it's the best animation app, period.

**Comment by MojoMan49** [comment]
There is an underlying elegance to what Keiwando is creating for sure. Far from public flogging I am a huge supporter.

I have been a user of Adobe Character Animator, sometimes combined w/ Premiere, but to get “pro” results is a really steep learning curve and hours of “puppet” making and an interface that is far from friendly.

I too want to thank Keiwando for all he has and continues to do!!
![EDDD071A-5153-42D4-A849-0235A5749D17](https://user-images.githubusercontent.com/127110520/229587020-3e9afc41-7c55-4067-9baa-69c5ecd1a11e.png)

**Comment by keiwando** [comment]
Thanks everyone, that's very kind!
